pirate/rgb.c

```diff
@@ -51,7 +64,8 @@ static uint offset;
     static const coordin8_t pixel_coordin8[] = {
         //                        // SIDE      POSITION    FACING
         // clang-format off
-        #if BP_REV >= 10
+        #if defined(RGB_HAS_ALL_PIXELS)
+        { .x =  90, .y = 255,  }, // bottom    left        side

```

Comment:
This is duplicated from the bottom and throws the assert error

Reply:
Wow!  I didn't really expect you to take all these changes as it wasn't my cleanest work (more intended for concepts / discussion).  That said, I'm glad they were useful enough for you to merge them in.

I've had problems building from the RP2350 branch.  I'll sync on the forum in case you have tips.   Even so, I cannot yet test on BP5XL/BP6 hardware; I'm eagerly awaiting the shipment!

Reply:
I did end up reversing the changes because it wouldn't build for rp2350. I cannot tell why, the changes all seemed simple enough.